examples/examples/z_storage.rs

```diff
@@ -46,9 +43,7 @@ async fn main() {
         .await
         .unwrap();
 
-    println!("Enter 'q' to quit...");
-    let mut stdin = async_std::io::stdin();
-    let mut input = [0u8];
+    println!("Press CTRL-C to quit...");
     loop {
         select!(
```

Comment:
Select is no longer necessary, just have a `while let Ok(sample) = subscriber.recv().await { .. }`.

Reply:
This one has a subscriber and a queryable, I believe a select is necessary.